Pieces.py

```diff
@@ -0,0 +1,33 @@
+class Piece:
+    location = (column, row)
```

Comment:
Whats the plan for this class property?

Reply:
In my mind, each piece has a location on the board at any given moment (or is removed from the board). So that would be an attribute, right?